src/utility.ts

```diff
@@ -194,11 +194,14 @@ export function isJavaExtEnabled(): boolean {
 
 export function isJavaExtActivated(): boolean {
     const javaExt = vscode.extensions.getExtension(JAVA_EXTENSION_ID);
-    return javaExt && javaExt.isActive;
+    if (javaExt && javaExt.isActive) {
+        return true;
+    }
+    return false;

```

Comment:
what's the problem with the old usage?

Reply:
`Type 'boolean | undefined' is not assignable to type 'boolean'`
How about `return !!javaExt && javaExt.isActive;` ?